components/mintmaker/staging/base/kustomization.yaml

```diff
@@ -12,9 +12,15 @@ images:
 - name: quay.io/konflux-ci/mintmaker
   newName: quay.io/konflux-ci/mintmaker
   newTag: 1f9c660c0bfba39d3d3c38eecc6bf26a68d0af4e
+- name: quay.io/konflux-ci/mintmaker-renovate-image
```

Comment:
It only updates the image when name matches, basically the image transformer follows the steps:
1. Find out all images, by default it only supports images listed in `containers` and `initcontainers`, so we extend it to support finding images in the specific annotation in https://github.com/redhat-appstudio/infra-deployments/pull/5054.
2. If any images found and `name` matches with the names in this images list, it will perform the transform for `newName`, `newTag` and `digest`.

kustomize doc for reference: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/transformerconfigs/README.md#images-transformer

Reply:
I see, thank you very much :+1: